### PR TITLE
Addressing issue https://github.com/litmuschaos/litmus/issues/1421

### PR DESCRIPTION
**Pull request by vijayto**
Addressing issues updated https://github.com/litmuschaos/litmus/issues/1421 for elves utils. Please take a look

### CODE REVIEW

**Comment by ksatchit** [comment]
Thank you for the PR @vijayto ! Will get back with the reviews shortly.

File: kubernetes/job/build.go [hunk 1]
```diff
@@ -135,3 +135,17 @@ func (b *Builder) Build() (*batchv1.Job, error) {
 	}
 	return b.job.object, nil
 }
+
+
+// WithAnnotations sets the annotations field of Job with provided value
+func (b *Builder) WithAnnotations(annotation map[string]string) *Builder {
+
```
Comment (chandankumar4) [inline kubernetes/job/build.go:149]: It's good if you add this check here
```
	if len(annotations) == 0 {
		b.errs = append(
			b.errs,
			errors.New("failed to build job object: missing annotations"),
		)
		return b
	}
```

File: kubernetes/pod/build.go [hunk 2]
```diff
@@ -135,3 +135,16 @@ func (b *Builder) Build() (*corev1.Pod, error) {
 	}
 	return b.pod.object, nil
 }
+
+
+// With Annotations - sets the Annotations field of Pod with provided value
+func (b *Builder) WithAnnotations(annotations map[string]string) *Builder {
```
Comment (chandankumar4) [inline kubernetes/pod/build.go:141]: Here also add this check:
```
	if len(annotations) == 0 {
		b.errs = append(
			b.errs,
			errors.New("failed to build pod object: missing annotations"),
		)
		return b
	}
```

**Comment by vijayto** [comment]
Thanks @chandankumar4 , think @ksatchit had suggested a few changes in operator and runner as well, let me update it , thanks again!

**Review by chandankumar4 — APPROVED** [review]
Thanks @vijayto For this PR

**Review by rahulchheda — APPROVED** [review]
/lgtm

**Review by ksatchit — APPROVED** [review]

**Pull request merged**